binderhub/tests/test_build.py

```diff
@@ -17,17 +17,17 @@
 
 @pytest.mark.asyncio(timeout=900)
 @pytest.mark.parametrize("slug", [
-    "gh/binderhub-ci-repos/requirements/d687a7f9e6946ab01ef2baa7bd6d5b73c6e904fd",
-    "git/{}/d687a7f9e6946ab01ef2baa7bd6d5b73c6e904fd".format(
-        quote("https://github.com/binderhub-ci-repos/requirements", safe='')
+    "gh/binder-examples/minimal-dockerfile/truly-minimal",
+    "git/{}/truly-minimal".format(
+        quote("https://github.com/binder-examples/minimal-dockerfile", safe='')
     ),
-    "git/{}/master".format(
-        quote("https://github.com/binderhub-ci-repos/requirements", safe='')
-    ),
-    "gl/minrk%2Fbinderhub-ci/0d4a217d40660efaa58761d8c6084e7cf5453cca",
+    "gl/consideRatio%2Fbinderhub-ci-repo/HEAD",
```

Comment:
Please don't change the repository and instead get your changes merged into `minrk/binderhub-ci` (or more ambitiously create a binderhub-ci org on Gitlab and lets use that)

Reply:
I created a new organization and invited a few people to it!